src/bot.js

```diff
@@ -192,22 +192,14 @@ const onPrivateMessage = async (message) => {
     } catch (e) {
       console.log(e);
     }
-  }
-  else if (commandCheck === "reply") {
-    //const destinationChannelId = content.split(" ")[1];
+  } else if (commandCheck === "reply") {
+    const destinationChannelId = content.split(" ")[1];
     const messageReplyId = content.split(" ")[2];
 
     const newContent = content.split(" ").slice(3).join(" ");
 
     try {
-      const channels = await client.channels
-      const channel = await channels.fetch( async (channel) => {
-        const foundChannel = channel.find(({ messages }) => messages.id === messageReplyId);
-        return foundChannel.id;
-      })
-      console.log(channel); //.messages.fetch(messageReplyId)
-      //const tests = await channel.find(({ messages }) => messages.id === messageReplyId)
-      //console.log(tests);
```

Comment:
I removed this test part because I cannot figure out how to search in all the channel messages and not only in the last 100 messages. So for now it is using `reply _channel_where_is_message_to_reply_Id_ message_to_reply_Id_ text`